NachoSpigot-Server/src/main/java/org/bukkit/craftbukkit/block/CraftBlockState.java

```diff
@@ -132,7 +129,7 @@ public int getTypeId() {
     }
 
     public byte getLightLevel() {
-        return light;
+        return getBlock().getLightLevel();
```

Comment:
If this is called multiple times, this is slower, because it has to low the block in the world, wouldn't suggest this change
Source: 
` public Block getBlock() {
        requirePlaced();
        return world.getBlockAt(x, y, z);
    }`

Reply:
Like this might be good if CraftBlockState gets generated everytime getblock is created and getLightLevel isn't hit alot